lib/core/terraform_config/generator.rb

```diff
@@ -13,6 +13,8 @@ def filename
       case template["kind"]
       when "gvc"
         "gvc.tf"
+      when "secret"
+        "secrets.tf"
```

Comment:
Small nit: You put secret below identity everywhere else, but here it comes above. I prefer to follow a consistent order.